.storybook/utils.ts

```diff
@@ -40,6 +40,26 @@ export const disableStorybookControls = <Props>(
 };
 const DISABLE_CONTROL = { control: false };
 
+/**
+ * Configure provided args to be listed under a specified
+ * category in the props table.
+ *
+ * Should be passed or spread to `argTypes`
+ */
+export const moveStorybookControlsToCategory = <Props>(
+  propNames: Array<keyof Props>,
+  category = 'Additional'
+): Record<keyof Props, ControlCategory> | {} => {
+  return propNames.reduce(
+    (obj, name) => ({ ...obj, [name]: setControlCategory(category) }),
```

Comment:
I'm totally good with this either way, but dev readability would you prefer to simply inline the object structure rather than call a fn? e.g.
```suggestion
    (obj, name) => ({ ...obj, [name]: { table: { category } } }),
```

Reply:
You're right. I tend to want to make things overly re-usable but if they are not re-used yet it makes no sense at that time. 😅